src/HttpHeader.cc

```diff
@@ -547,7 +549,7 @@ HttpHeader::parse(const char *header_start, size_t hdrLen, Http::ContentLengthIn
         // and clen state becomes irrelevant
 
         if (rawTe == "chunked") {
-            teChunked_ = true;
+            ; // NP: leave header present for chunked() method
```

Comment:
Nit: "NP" is not a common abbreviation. Most readers will need to look it up and/or will misinterpret it. IIRC, most of the existing Squid code misuses "NP" to mean "note please". AFAICT, the most common definition (outside of Squid) for NP is "no problem". We should avoid NP in any context (in favor of more common/understood variants). It is unnecessary here.

```suggestion
            ; // leave header present for chunked() method
```

Reply:
Interesting interpretation of "NP:". This is a en_NZ / en_AU 'ism - perhapse since we have other far more common phrases meaning "no problem". Accepting since the Squid documentation is being enforced to be en_US.